redis_session.c

```diff
@@ -490,6 +491,10 @@ PS_OPEN_FUNC(redis)
                                            persistent, persistent_id ? ZSTR_VAL(persistent_id) : NULL,
                                            retry_interval);
 
+            if (context  != NULL) {
```

Comment:
You need to initialise `context` with `NULL` before parsing params.

Reply:
done :). Thank you for your patience.

Reply:
This is not exactly what I asked but ok :) just add `context = NULL` after `redis_sock_set_stream_context`

Reply:
😂😅